apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/rubricSettingsModal.html.ts

```diff
@@ -278,35 +286,97 @@ export function RubricSettingsModal({ resLocals }: { resLocals: Record<string, a
   `;
 }
 
-function RubricItemRow(item: RubricData['rubric_items'][0] | null, index: number) {
+function RubricItemsWithIndent(rubric_items: RubricData['rubric_items'][0][] | null | undefined) {
+  if (!rubric_items) return unsafeHtml('');
+  const parentStack = [] as RubricData['rubric_items'][0][];
+  const itemRows = rubric_items.map((item) => {
+    // Find parent in stack and remove any items with deeper nesting than parent
+    const cutoffIdx = item.parent_id
+      ? parentStack.findIndex((i) => i.id === item.parent_id) + 1
+      : 0;
+    parentStack.splice(cutoffIdx, parentStack.length - cutoffIdx);
+
+    // Get child ids needed for ARIA mapping
+    const childIds = rubric_items
+      .filter((i) => i.parent_id === item.id)
+      .map((i) => 'rubric-item-' + i.number);
+
+    const parentId = parentStack.length > 0 ? parentStack[parentStack.length - 1].number : null;
+    // Generate HTML for current item
+    const result = RubricItemRow({
+      item,
+      index: item.number,
+      parent_index: parentId,
+      indent_level: parentStack.length,
+      child_idxs: childIds,
```

Comment:
Are these indexes or IDs? After converting `RubricItemRow` to use a object instead of individual arguments, it became clear there's some kind of mismatch here.